Projet/core/Actor.cpp

```diff
@@ -51,6 +51,13 @@ namespace Pitbull
 		}
 	}
 
+	void Actor::SpriteTick(const float ElapsedTime)
```

Comment:
No other solution than adding a sprite tick ?

Reply:
Haven't looked into it really, seems to be a problem according to Caro. Not a pretty fix, but I guess it will have to do since we're close to the deadline.